frontend/app/embed.ts

```diff
@@ -246,7 +260,7 @@ function init() {
         clearTimeout(t.delay);
         t.delay = null;
       }
-      t.events.forEach(event => t.node!.removeEventListener(event, t.animationStop, false));
+      t.events.forEach(event => t.node?.removeEventListener(event, t.animationStop, false));
```

Comment:
Pease revert `?`. After migrate `createInstance` api. t.node always exist and not equal `null`
l